libraries/AP_Notify/AP_Notify.cpp

```diff
@@ -44,31 +45,35 @@ AP_Notify *AP_Notify::_instance;
 #define TOSHIBA_LED_I2C_BUS_INTERNAL    0
 #define TOSHIBA_LED_I2C_BUS_EXTERNAL    1
 
+// all I2C_LEDS
+#define I2C_LEDS (Notify_LED_ToshibaLED_I2C_Internal | Notify_LED_ToshibaLED_I2C_External | \
```

Comment:
PCA9685 is also I2C, and you probably don't want to autoprobe it.... I'd rather see us continue to list them out per board, so it's easier to see exactly what's being loaded, and to add/remove as needed.